src/main/scala/intellij/haskell/external/repl/ProjectStackReplProcess.scala

```diff
@@ -47,20 +48,24 @@ private[repl] class ProjectStackReplProcess(project: Project) extends StackReplP
   }
 
   private def load(psiFile: PsiFile, moduleName: Option[String]): Option[(StackReplOutput, Boolean)] = {
-    val filePath = getFilePath(psiFile)
-    val moduleNameAction = ApplicationManager.getApplication.runReadAction(new Computable[Option[String]] {
-      override def compute(): Option[String] = {
-        moduleName.orElse(HaskellPsiUtil.findModuleName(psiFile))
+    if (!HaskellConsoleView.isConsole(psiFile)) {
```

Comment:
In #107 You also complained that

```
I also get this message in project repl:

12:04   [project-stack-repl] command: :load /λ Bla
12:04	[project-stack-repl] stdOut: ^IntellijHaskell^
11:50	[project-stack-repl] errOut: target ‘/λ’ is not a module name or a source file
```

Just the same reason here, so I add the `isConsole` predicate here 😄